go.mod

```diff
@@ -1,3 +1,26 @@
 module github.com/rollkit/avail-da
 
-go 1.21.0
+go 1.21.1
+
+toolchain go1.21.5
+
+require (
+	github.com/rollkit/go-da v0.0.0-20231024133951-57bc36006772
```

Comment:
go-da latest `v0.2.0` adds context and gas price for submit. Something to note for later.